.env

```diff
@@ -39,8 +39,27 @@ SECRET_KEY_BASE=56NtB48ear7+wMSf0IQuWDAAazhpb31qyc7GiyspBP2vh7t5zlCsF5QDv76chXeN
 INDEXER_DISABLE_PENDING_TRANSACTIONS_FETCHER=true
 ETHEREUM_JSONRPC_VARIANT=geth
 ETHEREUM_JSONRPC_HTTP_URL=http://kava:8545
+# uncomment below to have blockscout index public testnet
+# ETHEREUM_JSONRPC_HTTP_URL=https://evm.data-testnet.kava.io
+# ETHEREUM_JSONRPC_WS_URL=wss://wevm.data-testnet.kava.io
+# uncomment below to have blockscout index mainnet
+# ETHEREUM_JSONRPC_HTTP_URL=https://evm.data.kava.io
+# ETHEREUM_JSONRPC_WS_URL=wss://wevm.data.kava.io
+
 NETWORK= "Kava Ethereum Co-Chain"
 LOGO="/images/kava-logo.png"
 LOGO_TEXT="Ethereum Co-Chain"
 COIN=KAVA
 BLOCKSCOUT_VERSION=v5.1.5-kava
+
+# settings used to control blockscout indexer behavior and request rates
+# https://docs.blockscout.com/for-developers/information-and-settings/env-variables#indexer-management
+INDEXER_CATCHUP_BLOCKS_BATCH_SIZE=1
+INDEXER_CATCHUP_BLOCKS_CONCURRENCY=1
+INDEXER_INTERNAL_TRANSACTIONS_CONCURRENCY=1
+INDEXER_BLOCK_REWARD_CONCURRENCY=1
+INDEXER_RECEIPTS_CONCURRENCY=1
+INDEXER_COIN_BALANCES_CONCURRENCY=1
+INDEXER_CATCHUP_BLOCK_INTERVAL=5s
+# The block number, where import of blocks by catchup fetcher begins from.
+FIRST_BLOCK=0
```

Comment:
This should be 1?

Reply:
will test and document the result as part of the config of the deployed services

Reply:
for local 0 is correct because the dockerized kava node has / makes that block

Reply:
There is no block 0 in tendermint, the genesis block is height 1 and one of the differences between ethereum and cosmos based blockchains.

You are correct that it responds, but it should error.  It looks like a request for height zero actually responds with height 1 in etheremint, so probably a bug.

```
curl -sX POST -H "content-type: application/json" --data '{"jsonrpc":"2.0","method":"eth_getBlockByNumber","params":["1", true],"id":1}' https://evm.data.kava.io
```